Examples/Swift/ChangeWorldviewBoundariesExample.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+import Mapbox
+
+@objc(ChangeWorldviewBoundariesExample_Swift)
+
+class ChangeWorldviewBoundariesExample: UIViewController, MGLMapViewDelegate {
```

Comment:
The view controller is conforming to the `MGLMapViewDelegate` protocol but is not using any of its associated delegate methods (yet).